Orm/Xtensive.Orm/Orm/Logging/BaseLog.cs

```diff
@@ -61,63 +64,59 @@ public IDisposable InfoRegion(string message, params object[] parameters)
     /// </summary>
-    /// <param name="message">Message to write to.</param>
-    /// <param name="parameters">Values of parameters in <paramref name="message"/>.</param>
+    /// <param name="messageId">Message to write to.</param>
+    /// <param name="parameters">Values of parameters in <paramref name="messageId"/>.</param>
     /// <param name="exception">Exception, which must be written.</param>
-    public virtual void Info(string message, object[] parameters = null, Exception exception = null)
-    {
-      Write(new LogEventInfo(Name, LogLevel.Info, message, parameters, exception));
-    }
+    public virtual void Info(string messageId, object[] parameters = null, Exception exception = null) =>
+      Write(LogLevel.Info, messageId, parameters, exception);
 
     /// <summary>
     /// Writes warning message.
     /// </summary>
-    /// <param name="message">Message to write to.</param>
-    /// <param name="parameters">Values of parameters in <paramref name="message"/>.</param>
+    /// <param name="messageId">Message to write to.</param>
+    /// <param name="parameters">Values of parameters in <paramref name="messageId"/>.</param>
     /// <param name="exception">Exception, which must be written.</param>
-    public virtual void Warning(string message, object[] parameters = null, Exception exception = null)
-    {
-      Write(new LogEventInfo(Name, LogLevel.Warning, message, parameters, exception));
-    }
+    public virtual void Warning(string messageId, object[] parameters = null, Exception exception = null) =>
+      Write(LogLevel.Warning, messageId, parameters, exception);
 
     /// <summary>
     /// Writes error message.
     /// </summary>
-    /// <param name="message">Message to write to.</param>
-    /// <param name="parameters">Values of parameters in <paramref name="message"/>.</param>
+    /// <param name="messageId">Message to write to.</param>
+    /// <param name="parameters">Values of parameters in <paramref name="messageId"/>.</param>
     /// <param name="exception">Exception, which must be written.</param>
-    public virtual void Error(string message, object[] parameters = null, Exception exception = null)
-    {
-      Write(new LogEventInfo(Name, LogLevel.Error, message, parameters, exception));
-    }
+    public virtual void Error(string messageId, object[] parameters = null, Exception exception = null) =>
+      Write(LogLevel.Error, messageId, parameters, exception);
 
     /// <summary>
     /// Writes fatal error message.
     /// </summary>
     /// <param name="message">Message to write to.</param>
     /// <param name="parameters">Values of parameters in <paramref name="message"/>.</param>
     /// <param name="exception">Exception, which must be written.</param>
-    public virtual void FatalError(string message, object[] parameters = null, Exception exception = null)
+    public virtual void FatalError(string messageId, object[] parameters = null, Exception exception = null) =>
```

Comment:
The ```messageId``` parameter is unsynced with parameter description

Reply:
You can get copy of params description from suggested variants for other methods

Reply:
done